stories/message-page/message-page.stories.js

```diff
@@ -0,0 +1,212 @@
+        </div>
+    </div>
+</div>
+`;
+
+filter.storyName = 'Message Page with Filter';
+filter.parameters = {
+    docs: {
+        iframeHeight: messageBoxHeight,
+        storyDescription: `Use case: The user has set a filter and there are no results.
+
+Icon: <code class="docs-code">.sap-icon--filter</code>
+    `
+    }
+};
+
+export const search = () => `
+<div style="height: 600px; width: 100%;">
+    <div class="fd-message-page">
+        <div class="fd-message-page__container">
+            <div class="fd-message-page__icon-container">
+                <span class="sap-icon sap-icon--search fd-message-page__icon"></span>
+            </div>
+            <div class="fd-message-page__content">
+                <div class="fd-message-page__title">
+                    No matching items found.
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+`;
+
+search.storyName = 'Message Page with Search';
+search.parameters = {
+    docs: {
+        iframeHeight: messageBoxHeight,
+        storyDescription: `Use case: The user has searched for something but there are no results.
+
+Icon: <code class="docs-code">.sap-icon--search</code>
+    `
+    }
+};
+
+export const noItems = () => `
+<div style="height: 600px; width: 100%;">
+    <div class="fd-message-page">
+        <div class="fd-message-page__container">
+            <div class="fd-message-page__icon-container">
+                <span class="sap-icon sap-icon--product fd-message-page__icon"></span>
+            </div>
+            <div class="fd-message-page__content">
+                <div class="fd-message-page__title">
+                    No items are currently available.
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+`;
+
+noItems.storyName = 'Message Page with No Items';
+noItems.parameters = {
+    docs: {
+        iframeHeight: messageBoxHeight,
+        storyDescription: `Use case: The app contains no items.
+
+Icon: Show the product icon or an icon that matches your use case.
+    `
+    }
+};
+
+export const error = () => `
+<div style="height: 600px; width: 100%;">
+    <div class="fd-message-page">
+        <div class="fd-message-page__container">
+            <div class="fd-message-page__icon-container">
+                <span class="sap-icon sap-icon--document fd-message-page__icon"></span>
+            </div>
+            <div class="fd-message-page__content">
+                <div class="fd-message-page__title">
+                    Sorry, we can’t find this page.
+                </div>
+                <div class="fd-message-page__subtitle">
+                    <a href="#" class="fd-link" tabindex="0">Home Page</a>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+`;
+
+error.storyName = 'Message Page with an Error';
+error.parameters = {
+    docs: {
+        iframeHeight: messageBoxHeight,
+        storyDescription: `Use case: The app cannot show any content due to an error.
```

Comment:
Use the error message page for when the user attempts to view a page but an error has occurred, resulting in no available content. If you can, provide a link to the app start screen in the error message.

Mandatory: Add the <code>.sap-icon--document</code> class to the <code>fd-message-page___icon-container</code> to display a page icon.

Reply:
Mandatory: Use the <code class="docs-code">.sap-icon--document</code> icon.

I left this part like this. `.sap-icon--document` is inside the `fd-message-page___icon-container`, the class is not applied to it.